### PR TITLE
Input

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Input Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160275983-128fe07e-58f1-45e4-8027-651694f2d257.mp4

### CODE REVIEW

File: inputs/styles2-input.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/* Inputs Main */
+.input-container {
+  margin: 3rem;
+}
+
+.input-main-heading {
+  color: yellow;
```
Comment (i-am-a-codegirl) [inline inputs/styles2-input.css]: use var color

File: inputs/styles2-input.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  border: none;
+  border-bottom: 2px solid;
+}
+
+.password-input::placeholder {
+  font-weight: bold;
+  color: black;
+}
+
+/* border: 2px solid red;
```
Comment (i-am-a-codegirl) [inline inputs/styles2-input.css]: remove unwanted comment lines

File: inputs/styles2-input.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  border-bottom: 2px solid;
+}
+
+.error-password-input::placeholder {
+  font-weight: bold;
+  color: black;
+}
+
+.input-message {
+  margin-top: 0rem;
```
Comment (i-am-a-codegirl) [inline inputs/styles2-input.css:99]: you can just write 0 for margin-top: 0

**Review by i-am-a-codegirl — APPROVED** [review]
Code looks clean, just added some suggestions please check out.

File: grids/styles2-grids.css [hunk 4]
```diff
@@ -34,7 +34,7 @@
   border-radius: 0.5rem;
   padding: 1rem;
   background-color: #029aa8;
-  color: var(--ternary-color;
+  color: var(--ternary-color);
   font-size: 1.2rem;
   box-shadow: 3px 4px 5px #5e5e5e;
```
Comment (ravikumar0403) [inline grids/styles2-grids.css:39]: use var for this color as well

File: inputs/styles2-input.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
+/* Inputs Main */
+.input-container {
+  margin: 3rem;
+}
+
+.input-main-heading {
+  color: yellow;
+}
+.input-snippet iframe {
+  margin: 1rem 0rem;
```
Comment (ravikumar0403) [inline inputs/styles2-input.css]: If value is 0 then you can omit the units

File: inputs/styles2-input.css [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+.input-sub {
+  margin: 1rem 0rem;
+  color: white;
+  font-size: 1.2rem;
+}
+.input-snippet {
+  margin: 1rem 0rem;
+}
+.standard-input {
+  color: yellow;
```
Comment (ravikumar0403) [inline inputs/styles2-input.css]: use var for colors

**Review by ravikumar0403 — APPROVED** [review]
Looks good.. left a couple of sanity comments

**Pull request merged**